internal/cli/apps/app_registry.go

```
@@ -0,0 +1,537 @@
+	return nil
+}
+
+func validateUniqueASCResources(resources []asc.Resource[asc.AppAttributes]) error {
+	seen := make(map[string]struct{}, len(resources))
+	for _, resource := range resources {
+		appID := strings.TrimSpace(resource.ID)
+		if appID == "" {
+			continue
+		}
+		if _, ok := seen[appID]; ok {
+			return fmt.Errorf("App Store Connect returned duplicate app id %q", appID)
```

golangci-lint (2.11.4)

[error] ST1005: error strings should not be capitalized

(staticcheck)